app/models/workprofile.rb

```diff
@@ -1,3 +1,4 @@
 class Workprofile < ApplicationRecord
-    
+    belongs_to :industry
```

Comment:
what is industry  ? is this a totally new concept ? also I cant see the `industry` model